how-to/how-to-rename-field-in-content-type.md

```diff
@@ -0,0 +1,53 @@
+---
+title: How to rename a field in content type
+description: Rename a field in content type using mongodb '$rename' operator
+type: how-to
+page_title: Rename a field in a content type

```

Comment:
You could re-word it to include more ways of forming the question, e.g. How to Change the Name of a Field in a Content Type or something like that. Also, should have title-style casing, too.